print MRSIGNER & MRENCLAVE during build

Laisky opened the pull request:
It might be convenient to print out MRSIGNER & MRENCLAVE at build time

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/edgelesssys/edgelessdb?pullRequest=110) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/edgelesssys/edgelessdb?pullRequest=110) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/edgelesssys/edgelessdb?pullRequest=110) it.</sub>

thomasten commented:
Thanks for the contribution. How do you invoke the Docker build to see this output?

Laisky commented:
> Thanks for the contribution. How do you invoke the Docker build to see this output?

I changed the Dockerfile a bit to `docker build` it manually, so I can specific my own private.pem and some attributes.

but I found there is no place to get MRENCLAVE so I added a line to output. You can turn off this PR if you don't think it's necessary.

thomasten commented:
I think it's helpful. I just couldn't see the output. But I can see it if I add `--progress=plain` to the `docker build` command line.

By the way, in case you didn't know, you can specify your own private.pem as explained here https://github.com/edgelesssys/edgelessdb/blob/main/BUILD.md#build-the-docker-image
